best_practice_hw.sql

```diff
@@ -0,0 +1,19 @@
+/*
+* Author: ARIGA, young & energetic fresher
+* Purpose:SQL best practice training
+*/
+
+select 
```

Comment:
The keywords like `SELECT, FROM, WHERE...` should be written is UPPER case. Sorry for lack of documentation for it.